homeassistant/components/home_connect/api.py

```diff
@@ -77,6 +77,8 @@ def get_devices(self):
                 device = Dishwasher(self.hass, app)
             elif app.type == "FridgeFreezer":
                 device = FridgeFreezer(self.hass, app)
+            elif app.type == "Freezer":
```

Comment:
could it be combined with the `FridgeFreezer`, since the used methods are doing the same?

Reply:
Well, it could be combined, but implementing something like settings (#57491) would require changing it.